### ISSUE TITLE
Typo error in chapter 1 quiz

### ISSUE THREAD

**Issue by kamalpreetgrewal** [azat-co/react-quickly#31]
In chapter 1, the quiz question number 4:

    You must include react-com.js in order to render React elements on a web page.

The answer to this is true. I believe it should be true for **`react-dom.js`** instead of **`react-com.js`.**

**Comment by anthonylasso** [comment]
True, I believe it's a typo unless the question is accounting for both react-core and react-dom. Since you _can't_ use one to render without the other?
